android/source/VideoLocker/res/values-en/strings.xml

```diff
@@ -154,5 +154,10 @@
     <string name="facebook_text">Facebook</string>
     <string name="google_text">Google</string>
     <string name="course_not_listed_text">Don\'t see one of your courses?</string>
+    <string name="open_edx_btn_text">OPEN EDX.ORG IN MY BROWSER</string>
+    <string name="find_courses_dialog_text_1">Our team is currently working hard to let you enroll in courses within the app.</string>
+    <string name="find_courses_dialog_text_2">In the meantime,</string>
+    <string name="find_courses_dialog_text_3">you can enroll in courses through our website</string>
+    <string name="find_courses_dialog_text_4">- you\'ll need to log in, enroll, and then return to the app when you\'re done.</string>
```

Comment:
@shahidtamboli Please combine these 4 strings into single string.  Single message text shouldn't be split. This change applies to all translations of these strings.